lib/ansible/modules/windows/win_shortcut.ps1

```diff
@@ -26,7 +26,7 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
 
-$src = Get-AnsibleParam -obj $params -name "src" -type "path"
+$src = Get-AnsibleParam -obj $params -name "src"
```

Comment:
I actually thought about that on the PS declarative args stuff- accepting a list and trying to coerce down the list, but I haven't yet.